main.go

```diff
@@ -241,6 +245,44 @@ func visit(url *url.URL) {
 	}
 }
 
+// readResponseBody spools the http response body to disk or to screen depending
+// on the value of SOME FLAG.
+func readResponseBody(response io.Reader) string {
+	var err error
+	if showBodyEnv := os.Getenv("HTTPSTAT_SHOW_BODY"); showBodyEnv != "" {
+		showBody = getEnvOrDefault(showBodyEnv, showBody)
+	}
+
+	if showBody {
+		buf, _ := ioutil.ReadAll(response)
+		return fmt.Sprintf("RESPONSE BODY: %v", string(buf)) // @TODO limit buffer size
+	}
+
+	tmpfile, err := ioutil.TempFile("", "")
```

Comment:
This is the same logic as above, the only thing that changes is the destination that io.Copy is writing to.

It should be possible to write this function as
```go
var output = os.Stdout
if !showBody {
      // replace output with a temp file, defer its close
      fmt.Printf("RESPONSE BODY STORED IN: %v\n", tmpfile.Name())
}
io.Copy(output, response)
```